### PR TITLE
Added bxt_disable_skybox

### LINKED ISSUE #16: Skybox overrides view with bxt_wallhack 1
https://user-images.githubusercontent.com/58108407/127754721-72457173-3901-4848-98ea-9661c9103bf1.mp4

### PR DESCRIPTION
**Pull request by SmileyAG**
Closes #16

### CODE REVIEW

File: src/modules/disable_skybox.rs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        "bxt_disable_skybox"
+    }
+
+    fn cvars(&self) -> &'static [&'static CVar] {
+        static CVARS: &[&CVar] = &[&BXT_DISABLE_SKYBOX];
+        CVARS
+    }
+
+    fn is_enabled(&self, marker: MainThreadMarker) -> bool {
+        engine::R_DrawSkyBox.is_set(marker) && cvars::CVars.is_enabled(marker)
```
Comment (YaLTeR) [inline src/modules/disable_skybox.rs:21]: Also needs a `gl::GL.borrow(marker).is_some()`.

**Review by YaLTeR — COMMENTED** [review]
I'd investigate whether the issue can be fixed without disabling skybox (probably by moving skybox drawing earlier so it's drawn "below" everything else), but until then this workaround can stay.

File: src/modules/disable_skybox.rs (hunk 1 above)
Comment (SmileyAG) [reply src/modules/disable_skybox.rs]: Done.

**Comment by YaLTeR** [comment]
Thanks

**Pull request merged**